phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/MainWindowView.java

```diff
@@ -71,7 +74,7 @@ public void setCenter(Node nodeToShow) {
         this.content.setCenter(nodeToShow);
     }
 
-    public void showDetailsView(Node nodeToShow) {
+    public void showDetailsView(DetailsView nodeToShow) {
```

Comment:
I don't see a reason to restrict it in the software if it is not required. In #817 we had the same situation: `VBox` instead of `Node` for no reason, now I had to change it because I use a `StackPane`.